gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/impl/ReactableEvent.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.gateway.reactor.impl;
+
+import io.gravitee.definition.model.DefinitionVersion;
+import io.gravitee.gateway.reactor.ReactableApi;
+import java.util.Set;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/**
+ * @author Guillaume CUSNIEUX (guillaume.cusnieux at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@NoArgsConstructor
+@Setter
+@Getter
+@EqualsAndHashCode(callSuper = true)
+public class ReactableEvent<T> extends ReactableApi<T> {
```

Comment:
```suggestion
public class ReactableApiEvent<T> extends ReactableApi<T> {
```
The name `ReactableEvent` sounds too generic to extend from `ReactableApi` imho

Reply:
the event could be anything actually, I extends ReactableApi to get access to env/org id, enabled, deployedAt, no relation to API here.